src/org/opendatakit/briefcase/ui/BriefcaseCLI.java

```diff
@@ -85,9 +85,7 @@ public void run() {
       log.error("Error", e);
       System.exit(1);
     } catch (Throwable t) {
-      System.err.println("Unexpected error. Please review the logs and contact maintainers on the following URLs:");
-      System.err.println("\thttps://opendatakit.slack.com/messages/C374LNDK9/");
-      System.err.println("\thttps://forum.opendatakit.org/c/support");
+      System.err.println("Briefcase unexpected error. Please review briefcase.log for more information. For help, post to https://forum.opendatakit.org/c/support");
```

Comment:
I think the first part of this would read better as: "Unexpected error in Briefcase"

I think it'd also be able to provide a little more information about where the user can find briefcase.log. @ggalmazor  where is that file stored on the release builds?